src/goalServices/goals.js

```diff
@@ -518,6 +519,7 @@ export function reduceObjectivesForActivityReport(newObjectives, currentObjectiv
         'activityReportObjectiveResources',
         'resource',
         exists,
+        'value',
```

Comment:
A resource, for reasons that are obscure to me, has it's id renamed as value in the SQL query (I think this is also my fault), so this reducer was comparing undefined to undefined and therefore not returning all the resources.

This change allows us to configure which key we de-duplicate by